src/deepsparse/pipeline.py

```diff
@@ -175,6 +176,8 @@ def __init__(
 
         self.onnx_file_path = self.setup_onnx_file_path()
         self.engine = self._initialize_engine()
+        self._inference_timing = None
+        self.timer = TimingBuilder()
```

Comment:
timing builder's scope should just be within `call` not a property of the class (ie if there are multiple calls to call in async how might this affect the builder?)